src/dialects/postgres/schema/tablecompiler.js

```diff
@@ -49,8 +49,9 @@ TableCompiler_PG.prototype.comment = function(comment) {
 // Indexes:
 // -------
 
-TableCompiler_PG.prototype.primary = function(columns) {
-  this.pushQuery(`alter table ${this.tableName()} add primary key (${this.formatter.columnize(columns)})`);
+TableCompiler_PG.prototype.primary = function(columns, constraintName) {
```

Comment:
Yeah I know, but I try to keep the code the same across the project where possible, for consistency.

Could refactor all of these in a seperate PR I would suggest. This applies to several files ontop of tablecompiler.